docs/IDEOLOGIES.md

```diff
@@ -0,0 +1,24 @@
+# Ideologies
```

Comment:
Thinking about, I wonder if **Best Practice**  is better name?

Reply:
Yep much better 👍  thanks!